src/client/datascience/interactive-window/nativeInteractiveWindow.ts

```diff
@@ -400,7 +400,7 @@ export class NativeInteractiveWindow implements IInteractiveWindowLoadable {
             observable.subscribe(
                 (cells: ICell[]) => {
                     // Then send the combined output to the UI
-                    const converted = (cells[0].data as nbformat.ICodeCell).outputs.map(cellOutputToVSCCellOutput);
+                    const converted = (cells[0].data as nbformat.ICodeCell).outputs?.map(cellOutputToVSCCellOutput);
                     void temporaryExecution.replaceOutput(converted).then(() => {
```

Comment:
Does replaceOutput do nothing if converted is undefined?

Reply:
Good catch, updated to not perform a cast here, as that messes with TS's ability to flag problems like this